src/main/java/uk/gov/hmcts/reform/em/stitching/domain/DocumentImage.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.em.stitching.domain;
+
+import uk.gov.hmcts.reform.em.stitching.domain.enumeration.ImageRendering;
+import uk.gov.hmcts.reform.em.stitching.domain.enumeration.ImageRenderingLocation;
+
+public class DocumentImage {
+
+    private boolean enabled;
+    private String docmosisAssetId;
+    private ImageRenderingLocation imageRenderingLocation;
+    private Integer coordinateX;
```

Comment:
how come we're using boolean above and Integer here - unless you need it to be nullable int should be good enough